AlphaWallet/Core/Enjin/EnjinUserManagementInterceptor.swift

```diff
@@ -213,7 +213,7 @@ class UserManagementInterceptor: ApolloInterceptor {
                 } else if promise.isPending {
```

Comment:
It wasn't modified in the PR, but seems can we just store the promise and use `promise.done` multiple times, once for each "pending" block instead of manually maintaining a `Self.pending`?

Reply:
not clearly understood this